lib/screens/admin_new_product_page.dart

```diff
@@ -62,6 +63,9 @@ class _AdminNewProductPageState extends State<AdminNewProductPage> {
   Category _productCategory;
   StockType _productStockType;
 
+  List<CustomColor> _colorIterableList;
+  CustomColorList _customColorList;
+
```

Comment:
I sense some issue with the usage of these two variables. Why do we need to maintain a separate CustomColorList as well as a List<CustomColor>. I need some more time to take a deeper look, but if you can guide me thru your thought process, it would really help!

Reply:
`_colorIterableList` is used to create a List of color name and currently used in one place(**stock_selector.dart**).

Thank you for pointing it out. As for other places, we only need `_customColorList` to call item related method so I should remove the redundant code. I just updated!